linAssgn2

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+clear
+echo "HI :) " 
+find *.log > report.txt
+find *.txt >> report.txt
+tar -zcvf mathTest.tar.gz ../linux_assgn
```

Comment:
The requirement is to tar and compress only the *.txt and *.log from this folder. tar and compress should ignore all other files in this folder. Can you accomplish that?